src/Lesson2Hw.java

```diff
@@ -0,0 +1,73 @@
+public class Lesson2Hw {
+    public static void main(String[] args) {
+        System.out.println(compareTwoNumbers(2,18));
+        positiveOrNegative(-0);
+        System.out.println(positiveOrNegativeBoolean(-1));
+        printString(2,"Мир");
+        System.out.println(findLeapYear(400));
+
+    }
+
+    //1. Написать метод, принимающий на вход два целых числа и проверяющий,
+    // что их сумма лежит в пределах от 10 до 20 (включительно),
+    // если да – вернуть true, в противном случае – false.
+    private static boolean compareTwoNumbers(int a, int b) {
+        if ((10 <= (a+b)) && ((a+b)<=20)) {
```

Comment:
сумму можно вынести в отдельную переменную, чтобы не вычислять её дважды.
поскольку логическое выражение само по себе возвращает boolean результат, то можно его сразу же указать после слово return:
```suggestion
        return (10 <= (a+b)) && ((a+b)<=20);
```